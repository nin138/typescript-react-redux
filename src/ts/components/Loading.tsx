import * as styles from 'loaders.css/loaders.css';
import * as React from 'react';

const cubeColor1 = '#2196f3';
const cubeColor2 = '#ff96f3';
const cubeSize = '30px';
const cubeStyle = {
  width: cubeSize,
  height: cubeSize,
};
export const Loading = () => (
  <div
    className={styles['cube-transition']}
    style={{ display: 'inline-block', height: '1rem', transform: 'scale(.2)' }}>
    <div style={{ ...cubeStyle, background: cubeColor1 }} />
    <div style={{ ...cubeStyle, background: cubeColor2 }} />
  </div>
);
