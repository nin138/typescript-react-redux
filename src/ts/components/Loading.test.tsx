import * as React from 'react';
import renderer from 'react-test-renderer';
import { Loading } from './Loading';

it('Loading component', () => {
  const component = renderer.create(<Loading />);
  const tree = component.toJSON()!;
  expect(tree).toMatchSnapshot();
});
