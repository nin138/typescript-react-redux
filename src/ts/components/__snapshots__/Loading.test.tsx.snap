// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading component 1`] = `
<div
  style={
    Object {
      "display": "inline-block",
      "height": "1rem",
      "transform": "scale(.2)",
    }
  }
>
  <div
    style={
      Object {
        "background": "#2196f3",
        "height": "30px",
        "width": "30px",
      }
    }
  />
  <div
    style={
      Object {
        "background": "#ff96f3",
        "height": "30px",
        "width": "30px",
      }
    }
  />
</div>
`;
